pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -46,7 +46,6 @@ This section describes how to upgrade your Kommander Management cluster and all
   1. Pause the helm release.
   ```bash
   kubectl -n kommander patch -p='{"spec":{"suspend": true}}' --type=merge helmrelease/metallb
-  helmrelease.helm.toolkit.fluxcd.io/metallb patched
   ```
   ```sh
```

Comment:
If this is input, it should be bash
```suggestion
  ```bash
```

Reply:
@gvasquezvargas that is output